bolt/lib/Profile/BoltAddressTranslation.cpp

```diff
@@ -197,8 +197,10 @@ void BoltAddressTranslation::writeMaps(std::map<uint64_t, MapTy> &Maps,
             ? SecondaryEntryPointsMap[Address].size()
             : 0;
     if (Cold) {
-      size_t HotIndex =
-          std::distance(ColdPartSource.begin(), ColdPartSource.find(Address));
+      // `Maps` is keyed by output addresses.
```

Comment:
This comment seems self-explanatory. Maybe remove it?

Reply:
I'm fine with removing that comment. It's explained in multiple places including BAT documentation.